.build/cassandra-deps-template.xml

```diff
@@ -344,5 +340,13 @@
       <groupId>org.agrona</groupId>
       <artifactId>agrona</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.lucene</groupId>
+      <artifactId>lucene-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.lucene</groupId>
+      <artifactId>lucene-analyzers-common</artifactId>
+    </dependency>
```

Comment:
As before, we should send an email to the dev list about the new dependencies.

Reply:
As above, I have sent an email

Reply:
I guess we can remove the analyzers bit now?